beep/structure.py

```diff
@@ -436,7 +438,7 @@ def get_interpolated_diagnostic_cycles(self, diagnostic_available,
         group = diag_data.groupby(["cycle_index", "step_index", "step_index_counter"])
         incl_columns = ["current", "charge_capacity", "discharge_capacity",
                         "charge_energy", "discharge_energy", "internal_resistance",
-                        "temperature", "datetime_seconds"]
+                        "temperature", "datetime_seconds", "test_time"]
```

Comment:
Doesn't specifically matter for this PR, but how is the `test_time` field calculated? I recall we used to use it for computing `charge_time` and `time-temperature-integrated` features, and ran into some issues (negative values I think..).

Reply:
I think that the test_time for arbin can have some negative values right at the beginning of the test since the test_time is computed by the extraction script that I wrote. For the Maccor file this should never be the case since the test_time is output directly from the machine. In either case, we should really only be using this column for calculations requiring high precision, which should always take a delta in the time base and not the absolute value.